parkings/models/parking_area.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib.gis.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from parkings.models.mixins import TimestampedModelMixin, UUIDPrimaryKeyMixin
+
+
+class ParkingArea(TimestampedModelMixin, UUIDPrimaryKeyMixin):
+    '''
+    This is for whatever ID the external system that this parking lot was
+    imported from has assigned to this lot. There is no guarantee that it will
+    always be a number or that it will be unique, especially if multiple
+    sources are to be used.
+    '''
+    external_id = models.CharField(
+        max_length=32,
+        verbose_name=_('external ID'),
+        db_index=True,
+    )
```

Comment:
True, things should fail when we're trying to add the bad data, not when we accidentally get two results for the same query when we're expecting only one.